inc/notificationtargetticket.class.php

```diff
@@ -205,15 +205,15 @@ function addAdditionnalUserInfo(array $data) {
          return ['show_private' => 0];
       }
 
-      $query = "SELECT COUNT(*) AS cpt
-                FROM `glpi_profiles_users`
-                WHERE `users_id`='".$data['users_id']."' ".
-                      getEntitiesRestrictRequest("AND", "glpi_profiles_users", "entities_id",
-                                                 $this->getEntity(), true)."
-                      AND profiles_id IN (".implode(',', $this->private_profiles).")";
-      $result = $DB->query($query);
-
-      if ($DB->result($result, 0, 'cpt')) {
+      $result = $DB->request([
+         'COUNT'  => 'cpt',
+         'FROM'   => 'glpi_profiles_users',
+         'WHERE'  => [
+            'profiles_id'  => $this->private_profiles
+         ] + getEntitiesRestrictCriteria('glpi_profiles_users', 'entities_id', $this->getEntity(), true)
+      ])->next();
+
+      if ($result['cpt']) {
```

Comment:
`$result` could be `false` on edge cases.

Reply:
hum... ok, but this is certainly not the first time this is done like it.